### PR TITLE
🔀 Portal 추가

### PR DESCRIPTION
**Pull request by kangkong0805**
## 💡 개요
Portal component를 구현했습니다.

### CODE REVIEW

File: src/components/common/Portal/index.tsx [hunk 1]
```diff
@@ -0,0 +1,27 @@
+import { useState, ReactNode, useEffect } from 'react';
+import ReactDOM from 'react-dom';
+import { Positioner } from './style';
+
+type PortalProps = {
+  children: ReactNode;
+  onClose: () => void;
+};
```
Comment (jjaejjae12) [inline src/components/common/Portal/index.tsx:8]: 그냥 Props로 하는게 좋을것 같은데 어떻게 생각함?

Comment (kangkong0805) [reply src/components/common/Portal/index.tsx]: [확인](https://github.com/GSM-MSG/GAUTH-frontend/pull/73/commits/1206c3f38dea59f415ca28f21c10ce87df22e2c0)

File: src/components/common/Portal/style.ts [hunk 2]
```diff
@@ -0,0 +1,14 @@
...
+  height: 100vh;
+  position: fixed;
+  z-index: 99999;
+  background: rgba(0, 0, 0, 0.25);
+  display: flex;
+  justify-content: center;
+  align-items: center;
+  top: 0;
+  left: 0;
+`;
```
[the remark this answers is not in the document]
Comment (kangkong0805) [reply src/components/common/Portal/style.ts:14]: [해치웠다](https://github.com/GSM-MSG/GAUTH-frontend/pull/73/commits/1206c3f38dea59f415ca28f21c10ce87df22e2c0)

**Review by jjaejjae12 — APPROVED** [review]

File: src/components/MyProfilePage/index.tsx [hunk 3]
```diff
@@ -97,6 +100,16 @@ export default function MyProfilePage() {
...
+
+      {a && (
+        <Portal
+          onClose={() => {
+            setA(false);
+          }}
+        >
+          <div>fdsafdsaf</div>
+        </Portal>
+      )}
```
[the remark this answers is not in the document]
Comment (kangkong0805) [reply src/components/MyProfilePage/index.tsx:112]: 아 refector 안했네 ...

**Review by jjaejjae12 — APPROVED** [review]

File: src/components/MyProfilePage/index.tsx (hunk 3 above)
[the remark this answers is not in the document]
Comment (kangkong0805) [reply src/components/MyProfilePage/index.tsx]: [수](https://github.com/GSM-MSG/GAUTH-frontend/pull/73/commits/e8865447d0a3cd763af45065fa2aabbd619853ee)정[완](https://github.com/GSM-MSG/GAUTH-frontend/pull/73/commits/e8865447d0a3cd763af45065fa2aabbd619853ee)료

**Pull request merged**